### ISSUE TITLE
Error with Mask by Text node.

### ISSUE THREAD

**Issue by kerbi** [BadCafeCode/masquerade-nodes-comfyui#9]
Hi ! I'm getting this error whenever I want to use the Mask by Text node :

```
Error occurred when executing Mask By Text:

No module named 'clipseg.clipseg'; 'clipseg' is not a package

  File "F:\Stable Diffusion\comfyui\execution.py", line 151, in recursive_execute
    output_data, output_ui = get_output_data(obj, input_data_all)
  File "F:\Stable Diffusion\comfyui\execution.py", line 81, in get_output_data
    return_values = map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True)
  File "F:\Stable Diffusion\comfyui\execution.py", line 74, in map_node_over_list
    results.append(getattr(obj, func)(**slice_dict(input_data_all, i)))
  File "F:\Stable Diffusion\comfyui\custom_nodes\masquerade-nodes-comfyui\MaskNodes.py", line 136, in get_mask
    model = self.load_model()
  File "F:\Stable Diffusion\comfyui\custom_nodes\masquerade-nodes-comfyui\MaskNodes.py", line 183, in load_model
    from clipseg.clipseg import CLIPDensePredT
```

I've tried to manually install the clipseg and clip packages with pip in case it were missing, but no, it doesn't fix the issue.
I can do the import without errors in my virtual env, so I don't understand why it fails in the script.

Any idea ?

Thanks in advance and thanks for the nodes.

**Comment by manny5457** [comment]
Had the same error, search for a CLIPSeg node, and swap it for your old node and it should work. Did with mine.

**Comment by le-khang** [comment]
I have the same error. ClipSEG works as a great replacement but I would love to have the negative prompt in the node of BadCafeCode. How should I fix this issue please?
![2023-09-18 17_13_36-ComfyUI](https://github.com/BadCafeCode/masquerade-nodes-comfyui/assets/35536989/d40c8a12-e550-45a0-920d-848357e93538)
![2023-09-18 17_15_29-ComfyUI](https://github.com/BadCafeCode/masquerade-nodes-comfyui/assets/35536989/a82c8726-841b-4167-a8b5-0b022b93e111)

**Comment by hben35096** [comment]
Had the same error

**Comment by hben35096** [comment]
I think I've found a way to get close by adding "from clipseg.clipseg import CLIPDensePredT" to the top code in the MaskNodes.py file

import os
import torch
import numpy as np
import math
from torchvision import transforms
from torchvision.ops import masks_to_boxes
**from clipseg.clipseg import CLIPDensePredT**
import torchvision.transforms.functional as TF
import torch.nn.functional as torchfn
import subprocess
import sys

**Comment by Duemellon** [comment]
I tried that edit but did not resolve it.

**Comment by hben35096** [comment]
Did you install clipseg?

**Comment by April93** [comment]
I get this import error *only* when "[ComfyUI's ControlNet Auxiliary Preprocessors](https://github.com/Fannovel16/comfyui_controlnet_aux)" is installed/enabled. Disabling it allows Mask by Text to work properly. Seems there's an import conflict?

Using the alternative "clipseg" node isn't a proper alternative as it functions differently.

**Comment by bananasss00** [comment]
I fixed it this way:
1) raised the import to the top as hben35096 said https://github.com/BadCafeCode/masquerade-nodes-comfyui/issues/9#issuecomment-1743794447
2) renamed the extension masquerade-nodes-comfyui -> 1masquerade-nodes-comfyui (so it would load first)
3) no more error

**Comment by Phando** [comment]
Tried several of these solutions, still looking for a fix. Thanks for the great masks!

**Comment by merc4derp** [comment]
I've been having the same problem and I've pinpointed the issue for my installation at least, can't promise it will fix yours too.

The problematic node was clipseg, which is installed in the main ComfyUI\custom_nodes\ folder without a subfolder of its own.

Removing it through the manager (or simply deleting the clipseg.py file in the custom nodes folder) fixes masquerade.
